src/async_fuse/memfs/kv_engine/etcd_impl.rs

```diff
@@ -529,4 +1045,110 @@ mod test {
             assert_eq!(dir_entry.file_type(), FileType::Dir);
         }
     }
+
+    #[tokio::test]
+    async fn test_watch() {
+        let client = EtcdKVEngine::new_for_local_test(vec![ETCD_ADDRESS.to_owned()])
+            .await
+            .unwrap();
+        // Create a new key
+        let client_clone = client.clone();
+        tokio::spawn(async move {
```

Comment:
Should test the stream mode with a series of key create/update/delete

Reply:
Ok, I put it into a for range to test a series of operations.

Reply:
```bash
running 1 test
2024-06-20T05:28:01.272657Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:02.275220Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:03.277604Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: delete event, key=TEST_test_watch
2024-06-20T05:28:04.280747Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:05.283861Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:06.286420Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: delete event, key=TEST_test_watch
2024-06-20T05:28:07.290238Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:08.292962Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:09.294694Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: delete event, key=TEST_test_watch
2024-06-20T05:28:10.297038Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:11.299703Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:12.302634Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: delete event, key=TEST_test_watch
2024-06-20T05:28:13.305388Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:14.307972Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: put event, key=TEST_test_watch
2024-06-20T05:28:15.309519Z  INFO datenlord::async_fuse::memfs::kv_engine::etcd_impl: delete event, key=TEST_test_watch
test async_fuse::memfs::kv_engine::etcd_impl::test::test_watch ... ok
```